test/common/protobuf/utility_test.cc

```diff
@@ -33,6 +33,83 @@ TEST_F(ProtobufUtilityTest, convertPercentNaN) {
                EnvoyException);
 }
 
+namespace ProtobufPercentHelper {
+
+TEST_F(ProtobufUtilityTest, evaluateFractionalPercent) {
```

Comment:
I would also add a test to verify the uint64 -> uint32 logic you have in the other file.

Reply:
added one